slasher/detection/attestations/spanner.go

```diff
@@ -23,7 +19,7 @@ var _ = iface.SpanDetector(&SpanDetector{})
 // spans from validators and attestation data roots.
 type SpanDetector struct {
 	// Slice of epochs for valindex => min-max span + double vote filter
-	spans []map[uint64][3]uint16
+	spans []map[uint64]types.Span
```

Comment:
I'm not using a reference since the actual struct nearly the same size as the pointer would be, just an extra bool.